extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -37,21 +37,23 @@ const metadataProvider = classes.MetadataProvider;
 
 /**
  *
- * @param {string} name - Data source name
- * @param {string} wadoUriRoot - Legacy? (potentially unused/replaced)
- * @param {string} qidoRoot - Base URL to use for QIDO requests
- * @param {string} wadoRoot - Base URL to use for WADO requests
- * @param {boolean} qidoSupportsIncludeField - Whether QIDO supports the "Include" option to request additional fields in response
- * @param {string} imageRengering - wadors | ? (unsure of where/how this is used)
- * @param {string} thumbnailRendering - wadors | ? (unsure of where/how this is used)
+ * @param {string} baseUrl - The root URL for DICOMWeb calls
+ * @param {string} wadoURLPrefix - The URL prefix for WADO DICOMWeb calls
+ * @param {string} qidoURLPrefix - The URL prefix for QIDO DICOMWeb calls
+ * @param {string} stowURLPrefix - The URL prefix for STOW DICOMWeb calls
+ * @param {bool} qidoSupportsIncludeField - Whether QIDO supports the "Include" option to request additional fields in response
+ * @param {bool} enableStudyLazyLoad - Request series meta async instead of blocking
+ * @param {bool} supportsWildcard - Whether the server supports wildcards calls (i.e. DCM4CHEE)
  * @param {bool} supportsReject - Whether the server supports reject calls (i.e. DCM4CHEE)
- * @param {bool} lazyLoadStudy - "enableStudyLazyLoad"; Request series meta async instead of blocking
+ * @param {bool} staticWado - Wether the DICOMWeb client is a static wado one fot test purposes
  * @param {string|bool} singlepart - indicates of the retrieves can fetch singlepart.  Options are bulkdata, video, image or boolean true
  */
 function createDicomWebApi(dicomWebConfig, UserAuthenticationService) {
   const {
-    qidoRoot,
```

Comment:
Just extract the ...options from the parameter, so that you can pass any additional options in as required.